torch/csrc/jit/init.cpp

```diff
@@ -428,6 +428,8 @@ void initJITBindings(PyObject* module) {
              using namespace torch::jit::tensorexpr;
 	     return GetTECudaPointwiseBlockSize() = block_size;
 	   })
+      .def(
+	   "_jit_set_texpr_fuser_enabled", &torch::jit::tensorexpr::SetTexprFuserEnabled)
```

Comment:
Probably want @ZolotukhinM 's opinion on the right knob to expose to the world, but seems fine to me.